src/main/scala/org/allenai/plugins/CoreRepositories.scala

```diff
@@ -1,53 +0,0 @@
-package org.allenai.plugins
-
-import bintray.BintrayKeys._
-import sbt._
-import sbt.Keys._
-
-object CoreRepositories {
-
-  /** Common licenses */
-  object Licenses {
-    val apache2 = ("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html"))
-  }
-
-  /** Common resolvers */
-  object Resolvers {
-    val ai2PrivateReleases = Resolver.bintrayRepo("allenai", "private")
-    val ai2PublicReleases = Resolver.bintrayRepo("allenai", "maven")
-
-    // needed for spray-json:
-    val spray = "spray".at("https://repo.spray.io/")
-
-    val typesafeReleases = "Typesafe Releases".at("https://repo.typesafe.com/typesafe/releases/")
-
-    /** Default set of resolvers that will be added via CoreSettings */
-    val defaults = Seq(
-      spray,
```

Comment:
Do we not need these other resolvers anymore?

Reply:
Spray and typesafe, no. And I couldn't find anything that used jcenter. It's trivial to add back, and if we need them I will be adding them in scholar most likely.